modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/BlockSearchRequest.scala

```diff
@@ -0,0 +1,12 @@
+package org.tessellation.rosetta.server.model
+
+case class BlockSearchRequest(
```

Comment:
is it autogenerated as well?

Reply:
This is an intermediary class to hold the values from the Request that is relevant to our network.

Reply:
imo it should be similar to search request provided by rosetta api (example: operator as enum) instead of custom fields like here

Reply:
`isOr` and `isAnd` have been replaced by `operation: BlockSearchRequestOperation` where `BlockSearchRequestOperation` is an internal version of the operator enum.

Reply:
why `Operation`? it should be an `Operator`. and why there is a need for internal version of it, if Operator supports only `and` and `or` and there will be no different operators?

Reply:
I've changed the enum to `BlockSearchRequestOperator`. The one I made is for internal use so that we don't have to tie. our code base to an enum specified by a external entity.

Reply:
but you will use it also for other requests, so it's not exclusively for BlockSearchRequest.
You can name it `Operator` as you will map it one way or another to rosetta operator, so it should be corresponding (isomorphic)

Reply:
It has been renamed.